invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -366,13 +366,11 @@ <h2 id="files-heading">{{ _('Media files') }}</h2>
             {# Jump up button #}
             {%- block jump -%}
               <div class="ui grid">
-                <div class="jump-to-top centered row rel-mt-1" id="jump-btn">
-                  <a style="cursor: pointer;">
-                    <button class="ui button labeled icon" aria-label="{{ _('Jump to top of page') }}">
-                      <i class="arrow alternate circle up outline icon"></i>
-                      {{ _("Jump up") }}
-                    </button>
-                  </a>
+                <div class="centered row rel-mt-1">
+                  <button id="jump-btn" class="jump-to-top ui button labeled icon" aria-label="{{ _('Jump to top of page') }}">
```

Comment:
minor: do we always have only one jump up button per page ? otherwise I wouldn't assign the js event to element id, since element id should be unique

Reply:
The js event is assigned to the class. (But we also just have one button per page.)